Dockerfile

```diff
@@ -1,21 +1,17 @@
-# ---- Base Node ----
-FROM node:22.14.0-bookworm-slim as base
+FROM node:22-alpine AS base
```

Comment:
🟥 please don't use alpine. glibc performs better and is generally more predictable in its behaviour than alpine replacement
🟥 it is important to pin specific image version, not only the semver major point